ui/src/components/userProfile/UserProfileDesktop.js

```diff
@@ -223,6 +243,7 @@ const UserProfileDesktop = (props) => {
                   md: {
                     borderBottom: '2px solid #0628A3',
```

Comment:
if you have repeated class like this, can you make it assigned to a variable like your `signedUpButton`? makes it cleaner

Reply:
i know you just added the color, but would be nice to have drive-by fixes and improve on our codebase slowly when u can do it